analyticsdataserver/settings/base.py

```diff
@@ -311,6 +311,10 @@
 DEFAULT_PAGE_SIZE = 25
 MAX_PAGE_SIZE = 100
 
+# Only used in a local developer tools script. Override in local.py
+LMS_BASE_URL = 'http://localhost:8000/'
```

Comment:
What do you think of removing these from `base.py` altogether and put them in `local.py` with default values and a comment to replace them? These settings should only ever be used in `generate_fake_course_data.py` which is only ever run in the local environment, so I think it makes that more explicit.